packages/@webex/plugin-meetings/src/statsAnalyzer/index.ts

```diff
@@ -416,6 +388,73 @@ export class StatsAnalyzer extends EventsScope {
       return;
     }
 
+    // Generate empty stats results
+    if (!this.statsResults[type]) {
```

Comment:
looks like you are setting it here , Do you think this is ok then creating a const and attaching it !?

Reply:
Updated for a better approach